README.rst

```diff
@@ -288,6 +288,31 @@ This should yield the following output:
 Similar to the http version, a critical error will cause the command to quit with the exit code `1`.
 
 
+Advanced Configuration
```

Comment:
We have a `docs/settings.rst`. I feel this should rather go there, instead of the readme.